projects/ion/src/lib/no-data/no-data.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from '@angular/core';
+import { IconType } from '../core/types';
+
+@Component({
+  selector: 'ion-no-data',
+  templateUrl: './no-data.component.html',
+  styleUrls: ['./no-data.component.scss'],
+})
+export class IonNoDataComponent {
+  @Input() iconType?: IconType;
+  @Input() label: string;
```

Comment:
```suggestion
  @Input() label = "Não há dados";
```

I believe that we can add a default text here.